tests/group_by_test.py

```diff
@@ -376,10 +381,15 @@ def test_all_kind_of_aggs(
         # bugged in dask https://github.com/dask/dask/issues/11612
         # and modin lol https://github.com/modin-project/modin/issues/7414
         # and cudf https://github.com/rapidsai/cudf/issues/17649
-        request.applymarker(pytest.mark.xfail)
+        request.applymarker(
+            pytest.mark.xfail(
+                reason="Backend does not support multiple aggregations with the same column"
+            )
+        )
```

Comment:
can we remove the `reason` here please?

i'm keen on having links to github issues where possible, but i'm not sure how much adding `reason=` everywhere adds